examples/js/controls/OrbitControls.js

```diff
@@ -24,7 +24,8 @@
 			if ( domElement === undefined ) console.warn( 'THREE.OrbitControls: The second parameter "domElement" is now mandatory.' );
 			if ( domElement === document ) console.error( 'THREE.OrbitControls: "document" should not be used as the target "domElement". Please use "renderer.domElement" instead.' );
 			this.object = object;
-			this.domElement = domElement; // Set to false to disable this control
+			this.domElement = domElement;
+			this.domElement.style.touchAction = 'none'; // Set to false to disable this control
```

Comment:
This line is problematic.

Ideally controls set `touchAction` to `none` in `pointerdown` and undo the change in `pointerup` event listeners. Defining `touchAction` is necessary [to make pointermove work on mobile](https://stackoverflow.com/a/48254578/5250847).

`DragControls` and `TransformControls` do it this way. However, this approach does not work with `OrbitControls` since the class adds an event listener to `contextmenu`. It seems this listener prevents the usage of the above pattern since the `pointermove` event listener still aborts after a short period of time. Removing the `contextmenu` listener fixes the issue. This is however no appropriate solution since preventing the context menu is important for desktop.

Adding `this.domElement.style.touchAction = 'none';` in the ctor makes `OrbitControls` incompatible with `TransformControls` though since the latter one resets the `touchAction` value in its `pointerup` event listener. One possible solution is to set `touchAction` to `none` in the ctor of `OrbitControls`, `DragControls` and `TransformControls`.

Reply:
> One possible solution is to set touchAction to none in the ctor of OrbitControls, DragControls and TransformControls.

Added this change with the last commit 9a4099f720a7ff3f3c18716ec5fb0cc4f245f4cc

Reply:
Seems like the best solution to me 👍